lib/plugins/inspec-habitat/README.md

```diff
@@ -0,0 +1,149 @@
+# InSpec Habitat Plugin
+
+## Summary
+
+This plugin allows you to do the following:
+  1. Add Habitat configuration to a profile
+  2. Create/Upload a Habitat package from an InSpec profile
+
+Creating a [Habitat](https://www.habitat.sh/) package from an InSpec profile
+allows you to execute that profile as a service (via a Habitat Supervisor) on
+any Linux based platform.
+
+When running as a service, an InSpec report will be created in JSON format (by
+default at `/hab/svc/YOUR_SERVICE/logs/inspec_last_run.json`).  Additionally, a
+log of the last run can be located at
+`/hab/svc/YOUR_SERVICE/logs/inspec_last_run.json` and CLI output is viewable in
+the Supervisor logs by default. You can also configure this service to report
+to [Chef Automate](https://www.chef.io/automate/).
+
+See below for usage instructions.
+
+## Plugin Usage
+
+### Adding Habitat Configuration to an InSpec Profile
+
+Run the following command:
+
+```
+inspec habitat profile setup PATH
+```
+
+This will create the following files:
+  - habitat/plan.sh (Instructs Habitat on how to build the package)
+  - habitat/default.toml (Used to configure the running Habitat service)
+  - habitat/hooks/run (Shell script to execute this profile as a service)
+  - habitat/config/inspec_exec_config.json (JSON for `inspec exec` CLI options)
+
+### Creating a Habitat Package
+
+> This command requires Habitat to be installed and configured. For instructions
+on how to do that see [here](https://www.habitat.sh/docs/install-habitat/).
+
+Run the following command:
+
+```
+inspec habitat profile create PATH
+```
+
+This command will:
+  - Create a Habitat artifact (`.hart` file).
+
+> NOTE: If you are fetching packages from Chef Automate see
+[below](#Integrating-with-Chef-Automate).
+
+### Uploading a Habitat Package
+
+> This command requires Habitat to be installed and configured. For instructions
+on how to do that see [here](https://www.habitat.sh/docs/install-habitat/).
+
+Run the following command:
+
+```
+inspec habitat profile upload PATH
+```
+
+This command will:
+  - Create a Habitat artifact (`.hart` file).
```

Comment:
Does this always make a hart file?  Or only if one doesn't exist already?

Reply:
Will always create one since they are timestamped:

```
jerryaldrichiii-inspec-profile-hab_example-0.1.0-20190221211035-x86_64-linux.hart
```